scripts/exgfs_atmos_grib2_special_npoess.sh

```diff
@@ -109,6 +145,7 @@ done
 ################################################################
 export SHOUR=000
 export FHOUR=180
```

Comment:
Almost every time this job is run (for real, not just plumbing testing) the GFS long forecast will be longer than 180hrs but perhaps check if `FHMAX_GFS` is less than 180 and set `FHOUR=${FHMAX_GFS}` if so.

Reply:
@KateFriedman-NOAA Thank you. I added `export FHMAX_GFS=${FHMAX_GFS:-180}` (see line 53). I don't see where else this is used within the context of this `exgfs_atmos_grib2_special_npoess.sh`.

I apologize, but am I misunderstanding?

Reply:
`FHMAX_GFS` will be set so you don't need to check and add a default. What I was talking about was making sure that `FHMAX_GFS` is equal to or greater than 180 for this job. If someone runs this job with a forecast length that was less than 180 then this job would fail in theory. Suggest something like: if FHMAX_GFS<180 then FHOUR=$FHMAX_GFS

Reply:
```suggestion
export FHOUR=180
if (( FHOUR > FHMAX_GFS )); then
  export FHOUR=FHMAX_GFS
fi
```